### PR TITLE
Nc news 5

### PR DESCRIPTION
**Pull request by HalfElfAlistair**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,17 +1,46 @@
...
+          })
+  }, [])
+
+if (isLoading) {
+  return <div className="loading">
+            <p>Getting some data for you...</p>
+        </div>
+}
+
+return <div>
```
Comment (karlaferry) [inline src/App.js]: I would put this in an else block :)

File: src/components/QueryForm.jsx [hunk 2]
```diff
@@ -1,13 +1,34 @@
...
-            <select>
-                <option>all articles</option>
-                <option>coding</option>
-                <option>football</option>
-                <option>cooking</option>
-            </select>
+        <select onChange={(event) => {
+            setTopicSelect(event.target.value);
+            setRouteState(routeSelect(event.target.value));
+            event.preventDefault();
```
Comment (karlaferry) [inline src/components/QueryForm.jsx]: No need for a preventDefault here :)

**Review by karlaferry — APPROVED** [review]
Good job! Just left a couple of comments :)

File: src/App.js (hunk 1 above)
Comment (HalfElfAlistair) [reply src/App.js]: Ace thanks Karla, have done this now and will get cracking on the next ticket :)

Comment (HalfElfAlistair) [reply src/App.js]: Or I will once I've reviewed your other comment, my bad lol

**Pull request merged**